modules/actions.js

```diff
@@ -37,11 +38,43 @@ const saveDate = (event) => {
 
 };
 
-const saveLocation = (event) => {
+const saveSelectedLocation = (payload, respond) => {
+  const userId = payload.user.id;
+
+  switch(payload.actions[0].name) {
```

Comment:
Can we do some good indentation here. The `case` should not be on the same level as `switch`. These some [eslint rule](https://eslint.org/docs/rules/indent#switchcase), include it if it's not there.

Reply:
Sure thing 👍